Scala/Scala.sublime-syntax

```diff
@@ -1376,6 +1376,11 @@ contexts:
       set: single-type-expression-tail
     - match: (?=@{{plainid}})
       set: single-type-expression-tail
+    - match: \(
+      set:
+        - match: \)
```

Comment:
Shouldn't this be scoped with some variation of a `punctuation` scope?

Reply:
Yes!